thirdparty/README.md

```diff
@@ -172,7 +172,7 @@ Files extracted from upstream source:
 ## embree
 
 - Upstream: https://github.com/embree/embree
-- Version: 3.13.5 (698442324ccddd11725fb8875275dc1384f7fb40, 2022)
+- Version: 4.3.1 (daa8de0e714e18ad5e5c9841b67c1950d9c91c51, 2024)
```

Comment:
I don't know why it's missing, but I'd add `LICENSE.txt` to the files we include from upstream.